app/west.yml

```diff
@@ -30,5 +38,88 @@ manifest:
           - edtt
           - trusted-firmware-m
           - sof
+    - name: bsim
```

Comment:
Will all of these need to be installed now when people build locally or in the cloud? I can imagine this might slow things down.

Reply:
If you look, the `group` for all of these is `babblesim`, and near the top of the file:

```
group-filter: [-babblesim]
```

Which disables it by default, and you can use a command line to enable it so `west update` pulls it in.